sql/migrations/2022-07-07-identifier-templates/down.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE bookbrainz.identifier_type
```

Comment:
do you have a guideline to always run migrations in a transaction, or only when there are multiple statements?

Reply:
We don't really have an SQL guideline other than the in-progress MB one, but I think this is a sane suggestion regardless of the fact that there's only one statement.
Who knows who is going to copy this file's structure in the future?